facdb/sql/dsny_electronicsdrop.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS _dsny_electronicsdrop;
+SELECT uid,
+    source,
+    CONCAT(dropoff_sitename, ' ', 'Electronics Drop-off Site') as facname,
+    number as addressnum,
+    street as streetname,
+    address,
```

Comment:
`streetname` is all caps, whereas the streename in `address` is in proper case.  Should we make these consistent now?  Or do we format all addresses later in FacDB?

Reply:
we do that later